internal/providers/terraform/aws/efs_file_system.go

```diff
@@ -10,34 +12,60 @@ func GetEFSFileSystemRegistryItem() *schema.RegistryItem {
 	return &schema.RegistryItem{
 		Name:  "aws_efs_file_system",
 		RFunc: NewEFSFileSystem,
+		Notes: []string{"In Terraform config the availability_zone_name field is required in this resource for correct results"},
```

Comment:
This note looks strange a little bit. I can't find information about it, but maybe we should set `Standard` zone as default? @alikhajeh1, @tim775 what do you think about it?

Reply:
The logic should be: if `availability_zone_name` is not null, use One Zone, else use Standard.

Reply:
Done